src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -209,13 +209,24 @@ internal async Task<IIdxContext> InteractAsync(string state = null, Cancellation
         /// <summary>
         /// Calls the Idx introspect endpoint to get remediation steps.
         /// </summary>
-        /// <param name="idxContext">The IDX context that was returned by the `interact()` call</param>
-        /// <param name="cancellationToken">The cancellation token</param>
-        /// <returns>The IdxResponse.</returns>
-        internal async Task<IIdxResponse> IntrospectAsync(IIdxContext idxContext, CancellationToken cancellationToken = default(CancellationToken))
+        /// <param name="idxContext">The context to introspect.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>Task{IIdxResponse}</returns>
+        protected async Task<IIdxResponse> IntrospectAsync(IIdxContext idxContext, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I think this should be public since it's needed by the self-hosted widget, right?

Reply:
I think I got around needing it to be public by extending the IdxClient.  I'll review more closely.